sipa/blueprints/usersuite.py

```diff
@@ -148,15 +148,32 @@ def contact():
                                       'cancel_to': url_for('.index')})
 
 
+@bp_usersuite.route("/subscribe", methods=['GET'])
+@login_required
+def subscribe():
+    result = subscribeStatusPage(current_app.config['STATUS_PAGE_ENDPOINT'],
+                                 current_user.mail.raw_value)
+    if result is None:
+        flash(gettext("Es ist ein Fehler aufgetreten!"), "error")
+    elif result:
+        flash(gettext("Du wurdest zur Status-Page hinzugefügt. Du bekommst eine E-Mail mit "
+                      "weiteren Details."), "success")
+    else:
+        flash(gettext("Du hast die Statuspage bereits abonniert."), "warning")
+
+    return redirect(url_for('.index'))
+
+
 def render_payment_details(details: PaymentDetails, months):
     return {
         gettext("Zahlungsempfänger"): details.recipient,
         gettext("Bank"): details.bank,
         gettext("IBAN"): details.iban,
         gettext("BIC"): details.bic,
         gettext("Verwendungszweck"): details.purpose,
-        gettext("Betrag"): format_currency(months * current_app.config['MEMBERSHIP_CONTRIBUTION'] / 100, 'EUR',
-                                           locale='de_DE')
+        gettext("Betrag"): format_currency(
+            months * current_app.config['MEMBERSHIP_CONTRIBUTION'] / 100, 'EUR',
+            locale='de_DE')
```

Comment:
Starting here you have lots of unrelated changes. please revert these, this makes `git blame` and PR review a confusing.
If you want to improve formatting that's alright, but better done just on `develop` but in any case in a separate commit. :)